py/misc/compress.py

```diff
@@ -3,38 +3,47 @@
 import csv
 from zipfile import ZipFile
 import time
+import datetime
 import logging
 import logging.handlers as handlers
+import loggings_setup
 
+class Compress:
```

Comment:
Wrting a class is a bit overkill, but okay. Class names should be nouns: Compress -> Compressor?